workflows/databases/expressionatlas.nf

```diff
@@ -1,29 +1,33 @@
-include { slack_message } from './workflows/utils/slack'
-include { slack_closure } from './workflows/utils/slack'
-
 process fetch_data {
-  queue datamover
+  queue 'datamover'
+  container ''
+  errorStrategy 'ignore'
 
   output:
-  path('*.tsv')
+  path('tsv_files')
 
-  script:
   """
-  BASE_DIR="/nfs/ftp/public/databases/microarray/data/atlas/experiments/"
-  find $BASE_DIR -type f -iname "*tpms.tsv" -or -iname "*analytics.tsv" -or -iname "*sdrf.tsv" > all_relevant_files
+  find /nfs/ftp/public/databases/microarray/data/atlas/experiments -type f -name '*tpms.tsv' -or -name '*analytics.tsv' -or -name '*sdrf.tsv' -or -name '*configuration.xml' > all_relevant_files || true
+
+  sed -i '/E-MTAB-4748/d' all_relevant_files
```

Comment:
This can all probably be replaced with something like 

```
find $path -type f .. | grep -v E-MTAB-4748 | xargs -I {} -P 10 cp {} tsv_files
```

and would be much faster I suspect.

Reply:
Also one change that is worth making is that instead of doing `grep -v E-MTAB-4748` it is better to just have it copy all files and then add a filter operator in between this process and the one that works the the files. The filter would check if the basename of a file is in a list of excluded experiments or not. This is because you can change how the filter works, add remove, etc experiments and it will not have to recopy any data. The process output is cached but operators are rerun each time. I've found this really useful when having to restart the pipeline.